### PR TITLE
Adds friendly error message when certifcates are not found

### PR DESCRIPTION
**Pull request by benlemasurier**

### CODE REVIEW

File: cmd/logtalez/main.go [hunk 1]
```diff
@@ -31,6 +32,14 @@ func main() {
 		log.Fatal("--clientcertpath is mandatory")
 	}
 
+	if _, err := os.Stat(*serverCertPathPtr); os.IsNotExist(err) {
+		log.Fatalf("server certificate not found: '%s'", *serverCertPathPtr)
```
Comment (mdlayher) [inline cmd/logtalez/main.go]: You should use `%q` in place of `'%s'`; it'll handle quoting for you.

File: cmd/logtalez/main.go [hunk 2]
```diff
@@ -31,6 +32,14 @@ func main() {
...
+	}
+
+	if _, err := os.Stat(*clientCertPathPtr); os.IsNotExist(err) {
+		log.Fatalf("client certificate not found: '%s'", *clientCertPathPtr)
```
Comment (mdlayher) [inline cmd/logtalez/main.go]: You should use `%q` in place of `'%s'`; it'll handle quoting for you.

File: cmd/logtalez/main.go (hunk 1 above)
Comment (aybabtme) [inline cmd/logtalez/main.go]: you'd also want to print the `err`

Comment (bryanl) [inline cmd/logtalez/main.go]: Would you need to print the `err` if you checked the err with `os.IsNotExist`?

Comment (aybabtme) [inline cmd/logtalez/main.go]: oh right

Comment (benlemasurier) [inline cmd/logtalez/main.go]: I'll just check for any stat() error and print it out.

**Comment by aybabtme** [comment]
:shipit:

**Comment by taotetek** [comment]
:thumbsup:

**Pull request merged**